env_template

```diff
@@ -0,0 +1,22 @@
+# cp env_template .env
+# all variables in .env file will be loaded into environment variables after application starts
+[base]
```

Comment:
[base] is not a comment line, and python could recognize it?

Reply:
that's all right, it's .env/ini file grammar，just mark a section.